app/assets/stylesheets/components/_map.scss

```diff
@@ -40,6 +41,20 @@
     }
   }
 
+  &--header {
+    width: 100%;
```

Comment:
I'm also guilty of this, but isn't BEM for element level classes meant to be underscore-prefixed?

Reply:
This is a modifier (for a map that's in the header, as opposed to a map header). Header probably isn't the best word. I can try and think of a better one. It is quite misleading!

Reply:
Ah I see what you mean, okay. If you're happy I can merge, unless you have any last minute changes?